v0/destinations/fb/transform.js

```diff
@@ -220,16 +220,11 @@ function processEventTypeGeneric(message, baseEvent, fbEventName) {
 }
 
 function responseBuilderSimple(message, payload, destination) {
-  const requestConfig = {
-    requestFormat: "FORM",
-    requestMethod: "POST"
-  };
-
   const { appID } = destination.Config;
 
   // "https://graph.facebook.com/v3.3/644758479345539/activities?access_token=644758479345539|748924e2713a7f04e0e72c37e336c2bd"
```

Comment:
Let's update this comment too

Reply:
Also, make sure that the data in it is invalid.

Reply:
Updated